lib/float/common.sail

```diff
@@ -89,4 +89,46 @@ function float_decompose(op) = {
 val      not : forall ('p : Bool). bool('p) -> bool(not('p))
 function not(b) = not_bool(b)
 
+val      is_lowest_one : forall 'n, 0 < 'n. bits('n) -> bool
+function is_lowest_one (op) = {
+  let is_one = op[0] == bitone;
+
+  is_one;
+}
```

Comment:
```suggestion
function is_lowest_one (op) = {
  op[0] == bitone;
}
```
Woulnd't that be simpler?

Reply:
I see, thanks for comments. Looks we can do even simpler like lamba, let me have a try and update it.

Reply:
Updated.